src/nonvisual/mousemanager.js

```diff
@@ -506,31 +506,23 @@ var MouseManager = Base.$extend({
         // Load the current state
         this._action = action;
         this.__event = event;
-        this._button = null;
```

Comment:
What if action is not "mouse-down" ? this._button will be undefined instead of null, can it cause any problem ?

Maybe let it be null, as before.

Reply:
_button represent the laste state of the mouse therefore it shouldn't be at null.  
If you wnat the current state you should look at btnRight/BtnLeft/btnMiddle